submit_cgap/submission.py

```diff
@@ -825,7 +837,12 @@ def upload_file_to_new_uuid(filename, schema_name, auth, **context_attributes):
 
     post_item = compute_file_post_data(filename=filename, context_attributes=context_attributes)
 
-    response = ff_utils.post_metadata(post_item=post_item, schema_name=schema_name, key=auth)
+    if DEBUG_PROTOCOL:
```

Comment:
The precise text "pragma: no cover" is recognized as a marker saying that the coverage tools should not worry that there is no test coverage for conditionals like this.

```suggestion
    if DEBUG_PROTOCOL:  # pragma: no cover
```